src/v/rpc/simple_protocol.cc

```diff
@@ -17,6 +17,12 @@
 #include <exception>
 
 namespace rpc {
+
+/*
+ * the size threshold above which a reply message will be compression.
```

Comment:
nit: "use compression" / "be compressed"

Reply:
fixed